engine/src/main/java/org/terasology/physics/bullet/BulletPhysics.java

```diff
@@ -429,31 +450,30 @@ public boolean hasCharacterCollider(EntityRef entity) {
         return entityRigidBodies.keySet().iterator();
     }
 
-    @Override
-    public void awakenArea(org.terasology.math.geom.Vector3f pos, float radius) {
-        Vector3f min = new Vector3f(VecMath.to(pos));
-        min.sub(new Vector3f(0.6f, 0.6f, 0.6f));
-        Vector3f max = new Vector3f(VecMath.to(pos));
-        max.add(new Vector3f(0.6f, 0.6f, 0.6f));
-        discreteDynamicsWorld.awakenRigidBodiesInArea(min, max);
-    }
-
     @Override
     public void awakenArea(Vector3fc pos, float radius) {
-        Vector3f min = new Vector3f(VecMath.to(pos));
-        min.sub(new Vector3f(0.6f, 0.6f, 0.6f));
-        Vector3f max = new Vector3f(VecMath.to(pos));
-        max.add(new Vector3f(0.6f, 0.6f, 0.6f));
-        discreteDynamicsWorld.awakenRigidBodiesInArea(min, max);
+        btPairCachingGhostObject ghost = new btPairCachingGhostObject();
+        btSphereShape shape = new btSphereShape(radius);
+        ghost.setCollisionShape(shape);
+        ghost.setWorldTransform(new Matrix4f().translationRotateScale(pos, new Quaternionf(), 1.0f));
+
+        discreteDynamicsWorld.addCollisionObject(ghost, (short) -1, (short) -1);
+        for (int i = 0; i < ghost.getNumOverlappingObjects(); ++i) {
+            btCollisionObject other = ghost.getOverlappingObject(i);
+            other.activate(true);
+        }
+        discreteDynamicsWorld.removeCollisionObject(ghost);
     }
 
     @Override
     public float getEpsilon() {
-        return BulletGlobals.SIMD_EPSILON;
+        //TODO: figure out how access this from libgdx
+        return 1.19209290e-07f;
```

Comment:
How does this relate with the constant in this class that's also about Epsilon, and in fact shares the _same_ name as the old version of the getter? Same thing? This seems a _very_ magic number